atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -40,11 +41,19 @@ def run(self):
         # Download the source code archive
         archive = download_url(self.url, self.workflow.source.workdir)
 
-        # Unpack the source code archive into the workdir
```

Comment:
Read $COMMIT_MESSAGE for an explanation for removing this.

@rcerven brought an interesting question on IRC though: if we are to revert this and go back to extracting the sources in the workdir, would it be possible for files, such as the Dockerfile, to be overwritten? In this case, is this desirable?

Reply:
I'd say that dockerfile shouldn't be replaced ever.